internal/templating/ingress/test-resources/result-custom-ingress1.yaml

```diff
@@ -26,6 +26,7 @@ metadata:
     lagoon.sh/buildType: branch
     lagoon.sh/environment: environment-with-really-really-reall-3fdb
     lagoon.sh/environmentType: production
+    lagoon.sh/primaryingress: "true"
```

Comment:
This adds a third casing type to the labels: all lowercase. Currently the other labels are a mix of kebab-case and camelCase. I wonder if it should use one of the other standards?

Reply:
Yeah, good spot. Missed this.

Reply:
Most of the labels are camelcase, only one or two are kebab case.

Not sure which one to standardise on though at this point. But camelcase has more than the other

Reply:
camelCase makes more sense IMO - I'd only suggest kebab if there's a logical separation between words?